workloads/files/workload-concurrent-jobs-with-configmaps-script-cm.yml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scale-ci-workload-script
+data:
+  run.sh: |
```

Comment:
Think the boilerplate for enabling/disabling pbench data collection is missing. Here is the example: https://github.com/openshift-scale/workloads/blob/master/workloads/files/workload-conformance-script-cm.yml.

Reply:
Done.